specs/WebResourceRequested-CustomScheme.md

```diff
@@ -341,6 +346,26 @@ interface ICoreWebView2CustomSchemeRegistration : IUnknown {
   HRESULT SetAllowedOrigins(
     [in] UINT32 allowedOriginsCount,
     [in] LPCWSTR* allowedOrigins);
+
+  // Set this to `true` if the URLs with this custom scheme will have an
+  // authority component (a host for custom schemes).
+  // eg: custom-scheme-with-authority://host/path - Set to `true`
+  //     custom-scheme-without-authority:path - Set to `talse`
+  // When this is set to `true`, the URLs with this scheme will be
+  // interpreted as they have a
+  // [scheme and host](https://html.spec.whatwg.org/multipage/origin.html#concept-origin-tuple) 
+  // origin similar to an http URL. Note that the port and user
+  // information are never included in the computation of origins for
+  // custom schemes. If you set `HasAuthorityComponent` to `true` and 
+  // use a URL with this scheme that does not have
+  // an authority component, the behavior of such URLs will be undefined.
```

Comment:
I prefer not to define the unsupported behavior in case Edge changes the URL parsing. The spec also puts two alternatives in this case. The unsupported behavior should only be used at own risk.

Reply:
Practically we still have the problem though if devs take a dependency on the unsupported behavior. That's why I was asking about surfacing errors. If they don't read this part of the docs they could happen to depend on it without knowing. We need to either have an error for this case or define and support the behavior.

Reply:
Defined the behavior, but specified that it is not recommended to rely on this behavior in case it changes in future.